### PR TITLE
Change namespace in gRPC template

### PR DESCRIPTION
**Pull request by shirhatti**
Change namespace on the Grpc service to stay consistent with conventions

### CODE REVIEW

**Review by JunTaoLuo — APPROVED** [review]
That looks better, let's wait for the template tests to confirm it works.

File: src/ProjectTemplates/Web.ProjectTemplates/content/GrpcService-CSharp/Services/GreeterService.cs [hunk 1]
```diff
@@ -4,8 +4,9 @@
 using System.Threading.Tasks;
 using Grpc.Core;
 using Microsoft.Extensions.Logging;
+using GrpcService_CSharp
```
Comment (JamesNK) [inline src/ProjectTemplates/Web.ProjectTemplates/content/GrpcService-CSharp/Services/GreeterService.cs:7]: Alphabetical order usings

File: src/ProjectTemplates/Web.ProjectTemplates/content/GrpcService-CSharp/Startup.cs [hunk 2]
```diff
@@ -7,6 +7,7 @@
 using Microsoft.AspNetCore.Http;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
+using GrpcService_CSharp.Services;
```
Comment (JamesNK) [inline src/ProjectTemplates/Web.ProjectTemplates/content/GrpcService-CSharp/Startup.cs:10]: Alphabetical order usings

**Review by JamesNK — CHANGES_REQUESTED** [review]

File: src/ProjectTemplates/Web.ProjectTemplates/content/GrpcService-CSharp/Services/GreeterService.cs (hunk 1 above)
Comment (JamesNK) [inline src/ProjectTemplates/Web.ProjectTemplates/content/GrpcService-CSharp/Services/GreeterService.cs:7]:
```suggestion
using GrpcService_CSharp;
using Microsoft.Extensions.Logging;
```

File: src/ProjectTemplates/Web.ProjectTemplates/content/GrpcService-CSharp/Startup.cs (hunk 2 above)
Comment (JamesNK) [reply src/ProjectTemplates/Web.ProjectTemplates/content/GrpcService-CSharp/Startup.cs:10]: Move above Microsoft usings and below system

Comment (shirhatti) [reply src/ProjectTemplates/Web.ProjectTemplates/content/GrpcService-CSharp/Startup.cs:10]: Doesn't the order end up depending on the user's project name?

Comment (JamesNK) [reply src/ProjectTemplates/Web.ProjectTemplates/content/GrpcService-CSharp/Startup.cs:10]: Hmm, good point.

File: src/ProjectTemplates/Web.ProjectTemplates/content/GrpcService-CSharp/Services/GreeterService.cs (hunk 1 above)
Comment (JamesNK) [inline src/ProjectTemplates/Web.ProjectTemplates/content/GrpcService-CSharp/Services/GreeterService.cs:7]:
```suggestion
using GrpcService_CSharp;
```

**Review by JamesNK — APPROVED** [review]

**Comment by shirhatti** [comment]
Is this good to merge? Do we have template tests?

**Comment by pranavkm** [comment]
Merge?

**Pull request merged**